scripts/generate_release_updates.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script that generates announcement mail for the release."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
```

Comment:
Here and elsewhere, add single newline above this one to separate it from the fileoverview.

@kevinlee12 @oppia/dev-workflow-team this should probably be a lint check?

Reply:
Done.

Reply:
@kevinlee12 @oppia/dev-workflow-team Please could you confirm whether this should be a lint check?

Reply:
Yes, I've went ahead and created #7830